spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+RSpec.describe 'User show', type: :feature do
+  before(:each) do
+    # rubocop: disable Layout/LineLength
+
+    @first_user = User.create(name: 'Tom',
+                              photo: 'https://www.thoughtco.com/thmb/0I-Uw-0rcc6MUzcZJauNGKR9JzA=/768x0/filters:no_upscale():max_bytes(150000):strip_icc():format(webp)/male-computer-programmer-using-laptop-at-desk-in-office-755650739-5c5bb32346e0fb0001f24d3d.jpg', bio: 'Teacher from Mexico.', posts_counter: 0)
+    # rubocop: enable Layout/LineLength
+
```

Comment:
- [OPTIONAL] it is always a good practice to make sure you fix all rubocop linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉